backend/minors/ARA.yaml

```diff
@@ -0,0 +1,60 @@
+---
+type: Minor
+name: Arabic Language
+code: ARA
+degree:
+urls:
+- https://ua.princeton.edu/fields-study/minors/arabic-language
+description: The Department of Near Eastern Studies offers a 
+  wide variety of undergraduate classes focused on the languages, 
+  histories and cultures of the Near East. It provides opportunities 
+  for those who plan to major in other disciplines to simultaneously 
+  obtain proficiency in Arabic and to deepen their knowledge of the 
+  history, politics, literature, religion and culture of the premodern 
+  and modern Near East.
+contacts:
+- type: Director of Undergraduate Studies
+  name: Michael A. Reynolds
+  email: mar123@princeton.edu
+req_list:
+- name: Language Courses
+  max_counted: 1
+  min_needed: 4
+  pdfs_allowed: 0
+  explanation: Six language courses, two or more of which must be beyond 
+    the second-year level.
+  req_list:
+  - name: Intermediate / Advanced Language
+    max_counted: 4
+    min_needed: 1
+    course_list:
+    - ARA 105
+    - ARA 107
+    - ARA 111
+    - ARA 113
+    - ARA 3*
+    - ARA 4*
+- name: NES or additional language
+  max_counted: 1
+  min_needed: 1
+  explanation: Near Eastern studies course (either an additional language class or a content class listed or cross-listed in Near Eastern studies)
+  pdfs_allowed: 0
+  course_list:
+  - NES *
+  - ARA 3*
+  - ARA 4*
+- name: Senior Departmental Examination
```

Comment:
For Senior Departmental Examination we should add Editors Note as well as no_req signaling that we don't use it since student's don't enroll in a numbered course for it. Example at bottom here: [SOC.yaml](https://github.com/HoagieClub/plan/blob/5286f5433194b8663957c91b62c698ee07528132/backend/majors/SOC.yaml#L69)